src/layouts/mod.rs

```diff
@@ -123,13 +123,17 @@ impl Layout {
         if index < 0 {
             index = layouts.len() as isize - 1;
         }
-        layouts[index as usize].clone()
+        layouts[index as usize]
     }
 }
 
-// TODO: Perhaps there is a more efficient way to impl FromStr using serde
```

Comment:
I removed this TODO because there is no way using serde. That would enforce a serialization format just for a few string... not great.

I could make a macro_rule if you want... :thinking: